ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -495,7 +507,10 @@ func resourceIBMPIInstanceRead(ctx context.Context, d *schema.ResourceData, meta
 		d.Set("min_virtual_cores", powervmdata.VirtualCores.Min)
 	}
 	d.Set(helpers.PIInstanceLicenseRepositoryCapacity, powervmdata.LicenseRepositoryCapacity)
-	d.Set(PIInstanceDeploymentType, powervmdata.DeploymentType)
+	if powervmdata.DeploymentType != "" {
```

Comment:
If the value is empty you need to set the attr as "". So if condition is not required here.

Reply:
Set them "" if not available